### ISSUE TITLE
GPU mem question

### ISSUE THREAD

**Issue by guduzxc** [feihuzhang/GANet#17]
I'd like to know the GPU mem. requirement of >=12 GB is claimed for for one GPU card or the total GPU cards？
 If I only have two 2080ti (each mem. is 11GB，total mem. is 22GB) ， can I run the training process successfully?

Thanks for your sharing!

**Comment by feihuzhang** [comment]
You can adjust the batchsize = 2 to run the training and get a baseline. But, if you hope to get a high accuracy, e.g. compete with others on KITTI benchmarks. You may need eight 11G-mem GPUs or four 22G-mem GPUs (this is what we use to get the pre-trained models).
Or you can also try a simple model (we will try to publish one later) which is memory efficient.

**Comment by guduzxc** [comment]
Thanks, I see.

**Comment by feihuzhang** [comment]
Another way, you can try to update gradient after multi-iterations. Say pytorch have the settings “step” or something to get a similar (large batch size) training effects by only update the solver after K iterations.
